src/App.tsx

```diff
@@ -26,70 +27,144 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
+// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  // Sort goods if needed
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods.sort(
+        (firstGood, secondGood) => firstGood.localeCompare(secondGood),
+      );
+      break;
+
+    case SortType.LENGTH:
+      visibleGoods.sort(
+        (firstGood, secondGood) => firstGood.length - secondGood.length,
+      );
+      break;
+
+    default:
```

Comment:
it is better to handle all cases from enum to avoid unexpected behavior in the future, - what if another developer passes this unusable value to your function, and forgets to check if this case is handled? So best practice is to handle this case also and just break, and do not leave space for mistakes